src/plugins/workspace/public/components/workspace_collaborators/workspace_collaborators.tsx

```diff
@@ -116,13 +117,23 @@ export const WorkspaceCollaborators = () => {
         ]}
         setMountPoint={application?.setAppRightControls}
       />
-      <EuiPanel>
-        <WorkspaceCollaboratorTable
-          permissionSettings={permissionSettings}
-          displayedCollaboratorTypes={displayedCollaboratorTypes}
-          handleSubmitPermissionSettings={handleSubmitPermissionSettings}
-        />
-      </EuiPanel>
+      <EuiFlexGroup direction="column">
+        <EuiFlexItem>
```

Comment:
```
      <WorkspaceCollaboratorPrivacySettingPanel
        permissionSettings={permissionSettings}
        handleSubmitPermissionSettings={handleSubmitPermissionSettings}
      />
      <EuiSpacer />
      <WorkspaceCollaboratorTable
        permissionSettings={permissionSettings}
        displayedCollaboratorTypes={displayedCollaboratorTypes}
        handleSubmitPermissionSettings={handleSubmitPermissionSettings}
      />
```it would be like this if not using flex
![Screenshot 2024-11-26 at 10 32 22](https://github.com/user-attachments/assets/f90c3e56-0850-4dcb-a57c-905edf4b0291)

Reply:
Try to wrap with a regular div.
```

      <div>
        <WorkspaceCollaboratorPrivacySettingPanel
          permissionSettings={permissionSettings}
          handleSubmitPermissionSettings={handleSubmitPermissionSettings}
        />
        <EuiSpacer />
        <EuiPanel>
          <WorkspaceCollaboratorTable
            permissionSettings={permissionSettings}
            displayedCollaboratorTypes={displayedCollaboratorTypes}
            handleSubmitPermissionSettings={handleSubmitPermissionSettings}
          />
        </EuiPanel>
      </div>
```
It won't using a horizontal layout.

Reply:
BTW, the Edit button seems not aligned with "Workspace privacy" title. Shall we fix it?
![image](https://github.com/user-attachments/assets/d64f8775-068c-452b-a9c5-0fbdcd0c176e)

Reply:
thanks